addons/mail/static/src/core/web/messaging_menu.xml

```diff
@@ -17,7 +17,7 @@
 </t>
 
 <t t-name="mail.MessagingMenu.content">
-    <div t-att-class="`${discussSystray.contentClass} o-mail-MessagingMenu`">
+    <div t-if="!(ui.isSmall and store.discuss.activeTab === 'main')" t-att-class="`${discussSystray.contentClass} o-mail-MessagingMenu`">
```

Comment:
The messaging menu header should not be limited to main tab. There's the "start a meeting" and the "new channel/message" button that are needed.

Reply:
there is a `!` so it is actual removing the Messaging menu when it is not used

Reply:
Oh ok I didn't saw the `!`. In general it's easier to read when you apply De Morgan laws (https://en.wikipedia.org/wiki/De_Morgan%27s_laws) so `!(A and B)` => `!A or !B`

Reply:
yes, i know the law and the reason why I want to do like this is because `!(ui.isSmall and store.discuss.activeTab === 'main')` seems to be more meaningful than `!ui.isSmall or store.discuss.activeTab !== 'main'` because there is only one case being excluded and it is meaningful.